domains.csv

```diff
@@ -0,0 +1,1348 @@
+Domain Name,Domain Type,Agency,City,State
```

Comment:
Domains.csv should be removed from version control. This needs to be pulled dynamically from its permalink each time, so that this repository does not have to keep getting updated. That permalink is:

https://github.com/GSA/data/raw/gh-pages/dotgov-domains/current-federal.csv